example/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": false,
+    "target": "es5",
```

Comment:
is this the target we want?

Reply:
Note that this is in the `example` directory, so this isn't our `tsconfig.json`.

We actually don't include a `target` in our `tsconfig.json` - I just went with the `tsdx` default.

It looks like if you don't include a `target`, TS just defaults to `ES3`:

https://www.typescriptlang.org/tsconfig#target

Reply:
However, yes, the `target` we used previously was `es5`.

Reply:
I just went ahead and removed the `example` directory altogether.